File: CloudFlow/src/ro/cloudSoft/cloudDoc/presentation/client/shared/model/report/CentralizatorPrezentaPerioadaReportModel.java
package ro.cloudSoft.cloudDoc.presentation.client.shared.model.report;

import java.util.List;

public class CentralizatorPrezentaPerioadaReportModel {

	private List<CentralizatorPrezentaPerioadaReportRowModel> rows;
	private Integer totalParticipariLevel0;
	private Integer totalParticipariLevel1;
	private Integer totalParticipariLevel2;
	private Integer totalParticipariLevel3;
	private Integer totalParticipariLevel3Plus;
	private Integer totalParticipariInAfaraNom;

	public List<CentralizatorPrezentaPerioadaReportRowModel> getRows() {
		return rows;
	}

	public void setRows(List<CentralizatorPrezentaPerioadaReportRowModel> rows) {
		this.rows = rows;
	}

	public Integer getTotalParticipariLevel0() {
		return totalParticipariLevel0;
	}

	public void setTotalParticipariLevel0(Integer totalParticipariLevel0) {
		this.totalParticipariLevel0 = totalParticipariLevel0;
	}

	public Integer getTotalParticipariLevel1() {
		return totalParticipariLevel1;
	}

	public void setTotalParticipariLevel1(Integer totalParticipariLevel1) {
		this.totalParticipariLevel1 = totalParticipariLevel1;
	}

	public Integer getTotalParticipariLevel2() {
		return totalParticipariLevel2;
	}

	public void setTotalParticipariLevel2(Integer totalParticipariLevel2) {
		this.totalParticipariLevel2 = totalParticipariLevel2;
	}

	public Integer getTotalParticipariLevel3() {
		return totalParticipariLevel3;
	}

	public void setTotalParticipariLevel3(Integer totalParticipariLevel3) {
		this.totalParticipariLevel3 = totalParticipariLevel3;
	}

	public Integer getTotalParticipariLevel3Plus() {
		return totalParticipariLevel3Plus;
	}

	public void setTotalParticipariLevel3Plus(Integer totalParticipariLevel3Plus) {
		this.totalParticipariLevel3Plus = totalParticipariLevel3Plus;
	}

	public Integer getTotalParticipariInAfaraNom() {
		return totalParticipariInAfaraNom;
	}

	public void setTotalParticipariInAfaraNom(Integer totalParticipariInAfaraNom) {
		this.totalParticipariInAfaraNom = totalParticipariInAfaraNom;
	}

}

File: CloudFlow/FrontEnd-App/src/app/shared/model/reports/centralizator-prezenta-perioada-report.model.ts
import { JsonObject, JsonProperty } from "json2typescript";
import { CentralizatorPrezentaPerioadaReportRowModel } from "./centralizator-prezenta-perioada-report-row.model";

@JsonObject
export class CentralizatorPrezentaPerioadaReportModel {
	
	@JsonProperty("rows", [CentralizatorPrezentaPerioadaReportRowModel])
    public rows: CentralizatorPrezentaPerioadaReportRowModel[] = [];
    
	@JsonProperty("totalParticipariLevel0", Number)
	public totalParticipariLevel0: number = null;
    
	@JsonProperty("totalParticipariLevel1", Number)
	public totalParticipariLevel1: number = null;
    
	@JsonProperty("totalParticipariLevel2", Number)
	public totalParticipariLevel2: number = null;
    
	@JsonProperty("totalParticipariLevel3", Number)
	public totalParticipariLevel3: number = null;
    
	@JsonProperty("totalParticipariLevel3Plus", Number)
	public totalParticipariLevel3Plus: number = null;
    
	@JsonProperty("totalParticipariInAfaraNom", Number)
	public totalParticipariInAfaraNom: number = null;
}